flask_jsonapiview/view.py

```diff
@@ -108,6 +108,8 @@ class ModelView(ApiView):
     model = None
     url_id_key = 'id'
 
+    nested = None
```

Comment:
Think it's worth keeping the check at https://github.com/4Catalyzer/flask-jsonapiview/pull/26/files#diff-7e2c0e4609fe4eecb154d2d2534b07d4R183, then?